VMware.CloudFoundation.CertificateManagement.psm1

```diff
@@ -53,6 +53,29 @@ Function Get-Password {
     return $password
 }
 
+function Restart-VcenterService {
```

Comment:
Could you please add description for the function and params?
Also, Invoke-RestartService -Service $serviceName -- Do you need any wait or polling ?

Reply:
it's an internal function, so i just add a description to it.  There's no wait or polling required.  the Invoke-RestartService will only return when operation is complete.

Reply:
address in 30a32aca1c5451e5a0829cfb001061f9b16d0042